content/Services/RHOBS/rules-and-alerting.md

```diff
@@ -4,7 +4,7 @@
 
 As explained in more details [here](README.md), RHOBS features a deployment of [Observatorium](../../Projects/Observability/observatorium.md).
 
-Through the Observatorium API, tenants are able to **write**, **read** and **delete** their own Prometheus [recording](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) and [alerting](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) rules via the [Observatorium Rules API](https://observatorium.io/docs/design/rules-api.md/).
+Through the Observatorium API, tenants are able to **create**, **read**, **update** and **delete** their own Prometheus [recording](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) and [alerting](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) rules via the [Observatorium Rules API](https://observatorium.io/docs/design/rules-api.md/).
```

Comment:
I always find it a bit confusing that we actually cannot delete rule, we can only update the rules to be empty. If I recall correctly, we only support `PUT` verb, right? Which technically means we always update rules, but not really create or delete them.

Maybe just worth mentioning below, i.e. if a tenant wants to delete the rules, they actually have to update it (`PUT`) to be empty rule, as we don't support `DELETE` verb.

Reply:
Yes, let me address these comments in another PR! 🙂